simphony/io/cuds_file.py

```diff
@@ -85,6 +85,58 @@ def close(self):
         """
         self._file.close()
 
+    def add_lattice(self, name, lattice, keywords=[]):
```

Comment:
Please set the default argument value of `keywords` to `None`. Using mutable objects for default argument values can lead to very unexpected results in python (see http://docs.python-guide.org/en/latest/writing/gotchas/)